includes/templates/admin/edit-story.php

```diff
@@ -29,7 +29,7 @@
 	die( '-1' );
 }
 
-global $post_type, $post_type_object, $post;
+global $post_type, $post_type_object, $post, $wp_meta_boxes;
```

Comment:
Why is this global needed? `$wp_meta_boxes`

Reply:
Hmm I'll check, might be unneeded.